riemann/tests/tx/test_tx.py

```diff
@@ -1345,6 +1470,37 @@ def test_witness_hash(self):
             transaction.witness_hash(),
             helpers.DCR['ser']['witnesses'][0]['hash'])
 
+    def test_get_script_code(self):
+        transaction = tx.DecredTx(
```

Comment:
Can we improve this with a dict on the self object created in setup, and `tx.DecredTx(**self.tx_args)` like in zcash?